### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Response Compression: NVDA is not reading error message in a user understandable manner.

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#181]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Google Chrome Version 68.0.3440.106
Screen Reader: NVDA

**WCAG ID Violated:** WCAG1.3.1

GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #C+E-H; #.NETWebsites; #Chrome; #NVDA; WCAG1.3.1

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Server and press enter
5.	Tab till Response Compression and press enter
6.	Tab till Select Directory control and press enter
7.	Tab till New Folder control and press enter and press tab key
8.	Validate whether screen reader is reading the error message properly

**Actual Result:**
NVDA is not reading error message in a user understandable manner. It is reading inappropriate content

**Expected Result:**
NVDA should read error message in a user understandable manner.

**User Impact:**
Screen Reader users finds it difficult in understanding the error message if it is reading non programmatically

**Attaching video for your reference:**
[Bug 181_NVDA_Error Message.pptx](https://github.com/Microsoft/IIS.WebManager/files/2335391/Bug.181_NVDA_Error.Message.pptx)

**Attaching screenshot for your reference:**
![bug 181_error message_nvda](https://user-images.githubusercontent.com/42638744/44841369-cf539780-ac60-11e8-9e95-386c90c16d38.png)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
This issue will be fixed when we use WAC SDK controls.

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
